frontend/public/locales/en/translation.json

```
@@ -554,5 +557,38 @@
         }
       }
     }
+  },
+  "videos": {
+    "dialogs": {
+      "tutorial": {
+        "title1": "Welcome on Tournesol 🌻",
```

```suggestion
        "title1": "Welcome to Tournesol 🌻",
```
"Welcome to ..." seems more used, but "Welcome on ..." works also in this case if you prefer it
https://english.stackexchange.com/questions/492088/difference-between-welcome-to-welcome-on-and-welcome-at